### ISSUE TITLE
unable to send debug log : Échec lors de l’envoi des journaux : IDBDatabase.transaction: Can't start a transaction on a closed database

### ISSUE THREAD

**Issue by Marc-marc-marc** [vector-im/element-web#16757]
### Description

using Element Web, I try to send debug log for #16755 but it fail with the error "IDBDatabase.transaction: Can't start a transaction on a closed database"

### Steps to reproduce

- while logged in, click on my nickname on the top left
- help and about menu
- send debug log
- copy/past the url https://github.com/vector-im/element-web/issues/16755
- clic send

Describe how what happens differs from what you expected.

log send :)

Logs being sent: no, that the issue itselft 

### Version information

- **Platform**: web

For the web app:

- **Browser**: Firefox 78.8.0esr (64 bits)
- **OS**: CentOS Linux release 7.9
- **URL**: app.element.io

**Comment by jryans** [comment]
@Marc-marc-marc Were you in a private window? If yes, I believe this is the same as https://github.com/vector-im/element-web/issues/13290.

**Comment by Marc-marc-marc** [comment]
no, i'm not in a private windows

**Comment by jryans** [comment]
@Marc-marc-marc Okay, are you out of disk space? For some reason, the browser won't allow us to write to storage. This is mostly like a browser, OS, or system specific issue, as Element has little control over this.

**Comment by mpeter50** [comment]
I see this issue is stuck. Can I respond to this issue instead of the issue author?
I have very similar error messages in the browser console (see below), and I suspect that this has to do something with others not being able to read my encrypted messages.

```
Error running command: saveToDeviceBatches InvalidStateError: IDBDatabase.transaction: Can't start a transaction on a closed database [65df6e7ba71eb3d2272a.worker.js:1:62653](https://app.element.io/65df6e7ba71eb3d2272a.worker.js)
17:17:19.374
sendToDevice failed InvalidStateError: IDBDatabase.transaction: Can't start a transaction on a closed database
    g indexeddb-remote-backend.ts:194
[rageshake.ts:64:12](webpack:///node_modules/matrix-react-sdk/src/rageshake/rageshake.ts)
17:17:19.374
encryptAndSendToDevices promises failed InvalidStateError: IDBDatabase.transaction: Can't start a transaction on a closed database
    g indexeddb-remote-backend.ts:194
[rageshake.ts:64:12](webpack:///node_modules/matrix-react-sdk/src/rageshake/rageshake.ts)
17:17:19.374
failed to encryptAndSendToDevices InvalidStateError: IDBDatabase.transaction: Can't start a transaction on a closed database
    g indexeddb-remote-backend.ts:194
[rageshake.ts:64:12](webpack:///node_modules/matrix-react-sdk/src/rageshake/rageshake.ts)
17:17:19.375
Failed to share megolm keys for CK7SaKkW6V/WWRZ7Ne8l66nljBhPwbQj9uOiZ0nIuOw in !hbHduJHsSfDXsVCGcq:matrix.org (slice 1/1) [rageshake.ts:64:12](webpack:///node_modules/matrix-react-sdk/src/rageshake/rageshake.ts)
17:17:19.375
Failed to ensure outbound session in !hbHduJHsSfDXsVCGcq:matrix.org InvalidStateError: IDBDatabase.transaction: Can't start a transaction on a closed database
    g indexeddb-remote-backend.ts:194
```

**Comment by richvdh** [comment]
@mpeter50's issue sounds like it would be improved by the fix to https://github.com/vector-im/element-web/issues/23792.

The OP has gone away so I'm going to close this.
